docker-compose.yml

```diff
@@ -18,6 +18,7 @@ web:
     - REDIS_LOCATION=redis:6379
     - MYSQL_ROOT_PASSWORD='docker'
     - MYSQL_DATABASE='olympia'
+    - PYTHONDONTWRITEBYTECODE=1
```

Comment:
Well, mainly because prior to the other changes, all of the ones it created for me were owned by root. Beyond that, they don't seem to make a significant difference in startup speed, so the less pollution of the FS outside of docker, the better.